getconf: add page (#9515)
tldr-pages/tldr: pages/linux/getconf.md

# getconf

> > getconf - get configuration values from your Linux system.

Reviewer: ```suggestion
> getconf - get configuration values from your Linux system.
> More information: https://manned.org/getconf.1.
```

- This change is [mandatory](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#layout).
- More information should be provided to allow user explore command in details when needed.

Reviewer 2: Teminology such as *mandatory* sounds intimidating. Perhaps would could rephrase that?

Reviewer: I don't know. But I always divide fixes into two categories: mandatory (required by style guide or specification) and option (all others).

> > 

Reviewer 3: Please add more information link.

> - List every configuration values available

Reviewer 3: ```suggestion
- List every configuration values available:
```

> - List every configuration values available

Reviewer: ```suggestion
- List [a]ll configuration values available:
```

- This change is optional - no corresponding style guide requirement exists.
- We often provide mnemonics for short options to permit users easy memorize them.

`getconf -a`

> - List the configuration values for a specific directory
> 
> `getconf -a {{directory}}` 

Reviewer 3: ```suggestion
- List the configuration values for a specific directory:

`getconf -a {{path/to/directory}}` 
```

Reviewer: Some additions to a suggestion above:

- This change is [mandatory](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).
- Path format should be standardized across all pages.

> - Check if your linux system is a 32- or 64-bit system

Reviewer 3: ```suggestion
- Check if your linux system is a 32- or 64-bit system:
```

`getconf LONG_BIT`

> - Check how many processes a user can run at once

Reviewer 3: ```suggestion
- Check how many processes a user can run at once:
```

> - Check how many processes a user can run at once

Reviewer: ```suggestion
- Check how many processes the current user can run at once:
```

- This change is partially [mandatory](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#layout).
- Mandatory: trailing colons should be always provided.
- Optional: it's better to specify whether we are talking about the current user or not.

`getconf CHILD_MAX`

> - List every configuration value and then find patterns with the grep command. i.e every value with MAX in it.

Reviewer: ```suggestion
- List every configuration value and then find patterns with the grep command. i.e every value with MAX in it:
```

Almost the same issue as above.

> - List every configuration value and then find patterns with the grep command. i.e every value with MAX in it.
> 
> `getconf -a | grep MAX`

Reviewer 3: I think this example is not necessary.
```suggestion
```
